mne/coreg.py

```diff
@@ -1223,3 +1227,586 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
                 F_mri_ras, 'ras', 'ras'),
             F_ras_mni, 'ras', 'mni_tal')
     _write_fs_xfm(fname_to, T_ras_mni['trans'], kind)
+
+
+class Coregistration(object):
+    """
+    Class for MRI<->MEG coregistration.
+
+    Parameters
+    ----------
+    info : instance of Info
+        The measurement info.
+    subject : str
+        Name of the subject the data are defined for.
+    subjects_dir : path-like
+        Path to MRI subjects directory.
+
+    Notes
+    -----
+    Internal computation quantities ``parameters`` are in units of (for X/Y/Z):
+    - ``parameters[:3]`` are in radians
+    - ``parameters[3:6]`` are in m
+    - ``paramteres[6:9]`` are in scale proportion
```

Comment:
I was also wrong about this, it's always 9 params (and are the x/y/z scale factors)

Reply:
can we rather have more meaning variable names than parameters? eg rotations, translations, scales ?